neuvector-enforcer.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: neuvector-enforcer
+  version: 5.3.2
+  epoch: 0
+  description: "NeuVector Enforcer"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - busybox
+      - ca-certificates
+      - consul
+      - ethtool
+      - iproute2
+      - iptables
+      - jq
```

Comment:
do we really require this jq at runtime?

Reply:
Yup :/